process/modules/converb_to_conj.py

```diff
@@ -0,0 +1,180 @@
+        CoNLL.write_doc2conll(parsed_data, output_file_name)
+        participle_data = pyconll.load_from_file("participle.conllu")
+        roots = udon2.Importer.from_stanza(parsed_data.to_dict()) 
+        root = roots[0]  
+        selection = root.select_having("feats", "VerbForm=Conv") # выбираем все узлы, содержащие деепричастие
+        for node in selection:
+            subtree = node.get_subtree_text().replace(node.form, '', 1).strip(',.')
+            data['converb'] = node.form
+            inf = False
+            for feat in node.parent.feats:
+                if feat[0] == 'VerbForm' and feat[1] == 'Inf':
+                    inf = True
+            if inf:
+                data['head'] = node.parent.parent.form
+            else:
+                data['head'] = node.parent.form
+            data['dep'] = subtree
+            head_sub = node.parent.get_subtree_text().split()
+            data['agent'] = ''
+            ag_word = ''
+            if len(node.parent.get_by('deprel', 'nsubj')) != 0:
+                agent = node.parent.get_by('deprel', 'nsubj')[0]
+                ag_word = agent.form
+                data['agent'] = agent.get_subtree_text()
+            elif len(node.parent.parent.get_by('deprel', 'nsubj')) != 0:
+                agent = node.parent.parent.get_by('deprel', 'nsubj')[0]
+                ag_word = agent.form
+                data['agent'] = agent.get_subtree_text()
+            data['ag_word'] = ag_word
+        return data
+
+    def chose_converb(self, token):
+        conv_list = morph.parse(token)
+        conv = morph.parse(token)[0]
+        for elem in conv_list:
+            if elem.tag.POS == 'GRND':
+                conv = elem
+        return conv
+    
+    def normal_form(self, token):
+        conv = self.chose_converb(token)
+        conv_norm_list = morph.parse(conv.normal_form)
+        conv_norm = conv_norm_list[0]
+        for elem in conv_norm_list:
+            if elem.tag.POS == 'INFN':
+                conv_norm = elem
+        return conv_norm
+    
+    def converb_rewrite(self, sentence, preproc_utils: PreprocessingUtils):
+        rewritten_sentence = ''
+        data = self.converb_parser(sentence, preproc_utils)
+        converb = data['converb']
+        head_string = data['head']
+        dep = data['dep'].split()
+        sentence_list = []
+        agent = data['agent'].split()
+        agent_str = ''
+        and_ = 'и'
+
+        for elem in sentence.split():
+            sentence_list.append(elem.strip(',.:?!«»"'))
+
+        if sentence_list.index(converb) > sentence_list.index(head_string): #порядок слов, в котором главная клауза предшествует зависимой
+            for t in sentence.split():
+                token = t.strip(',.!?«»"')
+                if token == 'не' and sentence_list[sentence_list.index(token)+1] == converb:
+                    pass
+                elif token == converb:
+                    head = morph.parse(head_string)[0]
```

Comment:
```suggestion
                    head = preproc_utils.morph.parse(head_string)[0]
```

Используем `morph` из `preproc_utils`.